content/20.treatments.md

```diff
@@ -571,4 +571,37 @@ Summarize the state of the neutralizing antibody approach.
+COVID-19 results in hyperinflammation that appears to contribute to mortality via lung damage, suggesting that immunosuppression may be a helpful approach to treatment [@doi:10.1016/S0140-6736(20)30628-0].
+While tocilizumab targets IL-6, several other inflammatory markers could be potential targets, including TNF-alpha. Inhibition of TNF-alpha by an inhibitor such as Etanercept has been previously suggested for treatment of SARS-CoV [@doi:10.1185/030079903125002757] and may be relevant for SARS-CoV-2 as well.
+Baricitinib and other small molecule inhibitors of the JAK kinase pathway also curtail the inflammatory response and have been suggested as potential options for SARS-CoV-2 infections [@doi:10.1016/S1473-3099(20)30132-8].
+Baricitinib in particular may be able to reduce the ability of SARS-CoV-2 to infect lung cells [@doi:10.1016/S0140-6736(20)30304-4].
+Clinical trials studying baricitinib in COVID-19 have already begun in the US and in Italy (how to cite? US: https://investor.lilly.com/news-releases/news-release-details/lilly-begins-clinical-testing-therapies-covid-19, Italy: https://clinicaltrials.gov/ct2/show/NCT04320277).
+Identification and targeting of further inflammatory markers that are relevant in SARS-CoV-2 infection may be of value for curtailing the inflammatory response and lung damage.
+
+Another approach is the development of antivirals, which could be broad-spectrum, specific to coronaviruses, or targeted to SARS-CoV-2.
+The potential for remdesivir as an antiviral has already been described in an above section. Development of new antivirals is complicated by the fact that none have yet been approved for human coronaviruses. Intriguing new options are emerging, however.
+Beta-D-N4-hydroxycytidine (NHC) is an orally bioavailable ribonucleotide analog showing broad-spectrum activity against RNA viruses, which may inhibit SARS-CoV-2 replication [@doi:10.1126/scitranslmed.abb5883].
+Various other antivirals are in development.
+Development of antivirals will be further facilitated as research reveals more information about the interaction of SARS-CoV-2 with the host cell and host cell genome, mechanisms of viral replication, mechanisms of viral assembly, and mechanisms of viral release to other cells; this can allow researchers to target specific stages and structures of the viral life cycle.
+
+Antibodies against viruses, also known as antiviral monoclonal antibodies, could be an alternative as well and are described in detail in an above section.
+The goal of antiviral antibodies is to neutralize viruses through either cell-killing activity or blocking of viral replication [@doi:10.1016/j.tim.2015.07.005].
+They may also engage the host immune response, encouraging the immune system to hone in on the virus.
+Given the cytokine storm that results from immune system activation in response to the virus, which has been implicated in worsening of the disease, a neutralizing antibody (nAb) may be preferable.
```

Comment:
Thanks for your comments! I think this is a great point - I brought this up in greater detail in the section discussing tocilizumab and other anti-inflammatory agents a few paragraphs above this. I just rearranged a few sentences to make the arguments for and against more clear, and to add in the citation you shared.

I didn't repeat it in the nAbs section to avoid redundancy, but let me know if you think it's worth including here too.

Reply:
its great 👍